vm/2.0/modules/ROOT/pages/vm-publish-listen.adoc

```diff
@@ -2,36 +2,102 @@
 :page-aliases: connectors::vm-consume.adoc, connectors::vm/vm-publish-listen.adoc
 
 
-The following example shows how to publish a message that is received by a different flow. To do this, use the `<vm:publish>` and `<vm:listener>` elements that point to the same queue.
+The following example illustrates how to publish a message that is received by a different flow. The first flow takes an incoming HTTP message payload, then the *VM Publish* operation publishes the message to a VM queue. In the second flow, the *VM Listener* source receives the message payload from the previous flow via the same VM queue. To do this, point both operations to the same queue which prevents message loss caused by sending messages to a queue that nobody is listening to.
```

Comment:
```suggestion
The following example illustrates how to publish a message that is received by a different flow:

* The first flow receives an incoming HTTP message payload, then the *VM Publish* operation publishes the message to a VM queue. 
* In the second flow, the *VM Listener* source receives the message payload from the previous flow using the same VM queue. +
To do this, point both operations to the same queue, which prevents message loss caused by sending messages to a queue that nobody is listening to.
```

Reply:
I don't know about saying "nobody is listening to." Maybe something like "...a queue with no active listener?"